e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,14 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-40442 - CTRL/CMD+SHIFT+.', () => {
```

Comment:
```suggestion
    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
```

Reply:
I am new, why we changed the ticket number? @saturninoabril

Reply:
Good question. Nothing has changed. `MM-40442` is still the ticket number while `MM-T4452` is the test case ID which we used to track/record status of the test itself.